### ISSUE TITLE
Stash is not showing untracked files

### ISSUE THREAD

**Issue by rsalmei** [sublimehq/sublime_merge#859]
**Problem description**

Git enables us to stash untracked files, but they are not easily seen in sublime merge.
I just find out that they are in the third parent commit, so they could easily be shown (GitUp does show them).
More information: https://stackoverflow.com/questions/12681529/in-git-is-there-a-way-to-show-untracked-stashed-files-without-applying-the-stas

**Preferred solution**

Sublime Merge could merge the diff of tracked files and the untracked files, thus showing the complete picture of a stash. The way it does now is misleading, not showing all the files that were actually stashed.

**Alternatives**

It could at least include a button or make it clear there are more files not shown there.

**Comment by dpjohnst** [comment]
Hi @rsalmei,

Thank you for sharing this feedback with the Sublime Merge team.

The topology you see here is actually how Git stores untracked files (as a separate ref). We aim to keep a 1-1 mapping between refs and their contents.

That being said, we could definitely do more to make this clear via labels and links. 
I'll be sharing this feedback with the team.

Thanks,
\- Dylan

**Comment by srbs** [comment]
see also: #314

**Comment by rsalmei** [comment]
I understand @dpjohnst, thank you.
But to me, that is an implementation detail, when I click on a stash, I'd like to see its contents, all of it like when I stashed them, regardless of how git stores them.

On sublime merge, when I look into my last stash I see:
<img width="1668" alt="image" src="https://user-images.githubusercontent.com/6652853/90059057-f32b0f00-dcb8-11ea-931b-81358b9ff57a.png">

On the other hand, when I look it with GitUp I see:
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/6652853/90059158-1a81dc00-dcb9-11ea-9c0f-3702d417c61e.png">

**Comment by mwean** [comment]
I completely agree. Tower also combines tracked and untracked files, and it's so much nicer to use. I don't think it's helpful to stick precisely to git internals at the expense of user experience.

**Comment by c17r** [comment]
As of 03-Mar-2021, git now allows `git stash show [-u|--include-untracked]`.  You can also set `stash.showIncludeUntracked` to `true` so it always shows them.

It would be nice if SublimeMerge did something better here. I nearly dropped a stash because SM showed me nothing and I almost forgot that it was nothing BUT untracked files. At least git cmd gives me an easy way to check but I hate leaving the smooth silkiness that is SM (yes, trying a little flattery here!)

**Comment by ibeladi-mrsool** [comment]
Any progress on this?

It is harder to review which stash have the files you added, we need to review them one by one by popping or applying, which is very troublesome in some cases with high number of changed files and long list of stashes

**Comment by PunchyRascal** [comment]
Curiously enough, even with `showIncludeUntracked = true`  Sublime merge does not show the untracked file in a stash, but `git stash show 0` does.
(Or maybe I misunderstood and this not curious at all?)

**Comment by pietervisser** [comment]
Is there any progress on this?

**Comment by wnagrodzki** [comment]
@dpjohnst Is there any estimated date for when you might start working on this issue?
